platform/contracts/admin/src/endpoints.rs

```diff
@@ -1,6 +1,6 @@
 use access_control::ContractOwnerAccess;
 use platform::{batch::Batch, contract::CodeId, response};
-#[cfg(feature = "cosmwasm-bindings")]
+#[cfg(feature = "contract")]
```

Comment:
The current module is already under `#[cfg(feature = "contract")]` so this is not needed

Reply:
Good point, that didn't cross my mind.